ipfs/defaults.go

```diff
@@ -0,0 +1,201 @@
+package ipfs
+
+import (
+	"os"
+
+	ipfscfg "github.com/ipfs/go-ipfs-config"
+	"github.com/ipfs/interface-go-ipfs-core/options"
+)
+
+// BootstrapPeers is a list of default bootstrap peers for IPFS network
+// TODO: Change this to LL defaults at some point
+var BootstrapPeers, _ = ipfscfg.DefaultBootstrapPeers()
+
+// DefaultFullNodeConfig provides default embedded IPFS configuration for FullNode
+func DefaultFullNodeConfig() (*ipfscfg.Config, error) {
+	identity, err := ipfscfg.CreateIdentity(os.Stdout, []options.KeyGenerateOption{
+		options.Key.Type(options.Ed25519Key),
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	var conf = &ipfscfg.Config{
+		// ed25519 PKI identity for p2p communication
+		// TODO: Any reason not reuse ed25519 key from consensus identity?
+		Identity: identity,
+		// List of libp2p peer addresses the node automatically connects on start
+		Bootstrap: ipfscfg.BootstrapPeerStrings(BootstrapPeers),
+		// Configure all the addresses the node listens to
+		Addresses: ipfscfg.Addresses{
+			// List of libp2p addresses to listen to
+			Swarm: []string{
+				// listen on all network interfaces.
+				// NOTE: We don't use IPv6 for now, but we might add support for it at some point.
+				"/ip4/0.0.0.0/tcp/4001",
```

Comment:
Should we use a different port here by default s.t. this node won't collide with a local running node?

Reply:
Ok. Although, I don't see any reason to run two IPFS nodes on the same machines

Reply:
Yeah, I also don't see a reason for that. Other than maybe it might interfere with a running node when developing on the same machine? But that is maybe not the best reason. What do you think?

Reply:
The good reason actually